Release/src/utilities/asyncrt_utils.cpp

```diff
@@ -756,9 +758,9 @@ utility::string_t datetime::to_string(date_format format) const
         throw std::out_of_range("The requested year exceeds the year 9999.");
     }
 
-    const int64_t epochAdjusted = static_cast<int64_t>(m_interval) - Jan11900InNT;
+    const int64_t epochAdjusted = static_cast<int64_t>(m_interval) - NtTo1900OffsetInterval;
     const int64_t secondsSince1900 = static_cast<int64_t>(epochAdjusted / _secondTicks); // convert to seconds
```

Comment:
Is this static cast required?

Reply:
Yes; interval_type is uint64_t meaning this will warn about signed/unsigned mismatches without the cast. (interval_type should probably be int64_t but I'm scared to change that)

Reply:
I mean the line below -- line 760/762.
```
const int64_t secondsSince1900 = static_cast<int64_t>(epochAdjusted / _secondTicks); // convert to seconds
```
Since `epochAdjusted` is already int64, the static cast should not be needed iiuc

Reply:
Hmmmm my 'usual arithmetic conversions paranoia' is showing. I guess that one can probably go now.

Reply:
https://github.com/Microsoft/cpprestsdk/commit/b7d3e8b900bc5813deb455c5fff18d9603539265